src/components/NoticeText/Pricing10DlcNoticeText.tsx

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+
+const Pricing10DlcNoticeText: React.FC = () => (
```

Comment:
Automatically no, intentionally. Since we don't currently have anywhere for clients to track their pricing, the thought for the immediate future from @politics-rewired/spoke-client was to use this notice to do that

It can be manually hidden for clients who ask via the `SHOW_10DLC_REGISTRATION_NOTICES` envvar